### ISSUE TITLE
[Request] Nix pills on cross-compilation

### ISSUE THREAD

**Issue by Mic92** [NixOS/nix-pills#74]
This article is awesome. https://matthewbauer.us/blog/beginners-guide-to-cross.html
Something similar in nix pills would be great!

cc @matthewbauer

**Comment by matthewbauer** [comment]
Yeah sounds doable! I may have some time to write a little bit this weekend.
